### PR TITLE
new PR for demo

### PR DESCRIPTION
**Pull request by ycao233**

### CODE REVIEW

File: recipe.txt [hunk 1]
```diff
@@ -8,6 +8,6 @@ Guacamole
 1/2 jalapeño, seeded and diced
 juice of 1/2 lime
 1 tsp. salt
-pepper
+2 tsp. pepper
 
 Combine the avocado, onion, tomato, garlic, cilantro, and jalapeño, mixing until avocado is smooth.  Season with salt and pepper.  Top with the lime juice.  Serve with chips.
```
Comment (mmitrik) [inline recipe.txt]: comment from PR on 13

File: dev.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+dev client
+dev is no longer empty web
+another
```
Comment (mmitrik) [inline dev.txt]: good change

File: master.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+now behind by 1
+client
```
Comment (mmitrik) [inline master.txt]: I might do this differently

File: recipe2.txt [hunk 4]
```diff
@@ -1,13 +1,34 @@
...
 1/4 c. chopped tomato
 1 clove minced garlic
 2 tbsp chopped cilantro
 1/2 jalapeño, seeded and diced
-juice of 1/2 lime
-1 tsp. salt
-pepper
+juice of 1 lime
+1 tbsp. salt
+2 tbsp. pepper
```
Comment (mmitrik) [inline recipe2.txt]: Is this enough? :8ball:

File: recipe2.txt [hunk 5]
```diff
@@ -1,13 +1,34 @@
...
+
+taylor
+
+MVPDemo
+dev
+master
+123
+feature2
+
+test tag
```
Comment (mmitrik) [inline recipe2.txt]: I like this approach

File: dev.txt (hunk 2 above)
Comment (InTheParasol) [inline dev.txt]: I tend to agree

File: master.txt (hunk 3 above)
Comment (InTheParasol) [inline master.txt]: Me too

File: recipe2.txt (hunk 5 above)
Comment (InTheParasol) [inline recipe2.txt]: I do not

File: dev.txt (hunk 2 above)
Comment (mmitrik) [inline dev.txt]: what would you do differently?

File: recipe2.txt (hunk 5 above)
Comment (mmitrik) [inline recipe2.txt]: tell me more

File: .gitattributes [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+#*.dbproj    merge=binary
+#*.fsproj    merge=binary
+#*.lsproj    merge=binary
+#*.wixproj   merge=binary
+#*.modelproj merge=binary
+#*.sqlproj   merge=binary
+#*.wwaproj   merge=binary
+
+###############################################################################
+# behavior for image files
```
Comment (InTheParasol) [inline .gitattributes]: a second review

File: dev.txt (hunk 2 above)
Comment (InTheParasol) [inline dev.txt]: part of my second review

File: recipe2.txt [hunk 7]
```diff
@@ -1,13 +1,34 @@
...
 1/4 c. chopped onion
 1/4 c. chopped tomato
 1 clove minced garlic
 2 tbsp chopped cilantro
 1/2 jalapeño, seeded and diced
-juice of 1/2 lime
-1 tsp. salt
-pepper
+juice of 1 lime
+1 tbsp. salt
```
Comment (InTheParasol) [inline recipe2.txt]: second review comment

File: recipe2.txt (hunk 4 above)
Comment (InTheParasol) [inline recipe2.txt]: Add more (second)

File: .gitattributes (hunk 6 above)
Comment (mmitrik) [inline .gitattributes]: third

File: dev.txt (hunk 2 above)
Comment (mmitrik) [inline dev.txt]: Third

File: recipe.txt [hunk 8]
```diff
@@ -8,6 +8,6 @@ Guacamole
 1/2 jalapeño, seeded and diced
 juice of 1/2 lime
 1 tsp. salt
-pepper
+2 tsp. pepper
```
Comment (mmitrik) [inline recipe.txt]: enough?

**Pull request closed without merging**